draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.tsx

```diff
@@ -105,7 +105,12 @@ export const VideoPlayer = ({
          */
       }
     }
-  }, [prefersReducedMotion])
+    /**
+     * Chrome seems to have an issue with changes to autoplay after the video
+     * has been mounted. If there is a change in autoplay we need to force the
+     * play() method to be called.
+     */
+  }, [prefersReducedMotion, autoplay])
 
   useEffect(() => {
     // Add event listeners for the video element
```

Comment:
A note here: 
The `videoRef` dependency in this `useEffect` hook is redundant because it's not the reference/box that changes, its the field `videoRef.current` that does.
The other thing is that this might not get re-run reliably because of how ref changes don't cause the component to get re-rendered.
There are a few ways to get around this, e.g https://reactjs.org/docs/refs-and-the-dom.html#callback-refs

Seeing as this isn't related to your PR i've just approved so it's up to you!

Reply:
This involves a little bit more refactoring than I accounted for - I'll create an issue and loop back to it. Thank you for raising 🙏🏻